src/pybind/mgr/dashboard/frontend/src/app/shared/directives/update-scope.directive.ts

```diff
@@ -0,0 +1,34 @@
+import { AfterViewInit, Directive, ElementRef, Input, Optional } from '@angular/core';
+import { Permissions } from '../models/permissions';
+import { AuthStorageService } from '../services/auth-storage.service';
+import { PermissionScopeDirective } from './permission-scope.directive';
+
+@Directive({
+  selector: 'input:not([cdNoUpdateScope]), select:not([cdNoUpdateScope]), [cdUpdateScope]'
```

Comment:
I don't think we should use a selector like this.
Currently this is being applied to all forms in the app, even if we don't want to disable anything.

Reply:
```suggestion
  selector: '[cdScope] input, [cdScope] select, [cdUpdateScope]'
```